docs/oscar_references.bib

```diff
@@ -558,3 +558,30 @@ @Book{Zie95
   doi           = {10.1007/978-1-4613-8431-1},
   url           = {https://doi.org/10.1007/978-1-4613-8431-1}
 }
+
+@Book{GLS07,
+  author        = {Greuel, G.-M. and Lossen, C. and Shustin, E.},
+  title         = {Introduction to Singularities and Deformations},
+  series        = {Springer Monographs in Mathematics},
+  publisher     = {Springer-Verlag, Berlin},
+  year          = {2007},
+  pages         = {xii+471}
+}
+
+@Book{dJP00,
+  author        = {de Jong, Theo and Pfister, Gerhard},
+  title         = {Local Analytic Geometry},
+  series        = {Advanced Lectures in Mathematics},
+  publisher     = {Vieweg+Teubner Verlag},
+  year          = {2000},
+  pages         = {xi+384}
+}
+
+@Book{Har77,
+  author        = {Hartshorne, Robin},
+  title         = {Algebraic Geometry},
+  series        = {Graduate Texts in Mathematics},
+  publisher     = {Springer-Verlag, New York},
+  year          = {1977},
+  pages         = {xvi+496}
+}
```

Comment:
Who is @afkafkafk13 (Is that Anne?) It is a bit annoying to have these completely anonymous accounts (that includes @HechtiDerLachs !) make comments here. Could you perhaps at least set a real name in your profile, so that it is shown upon mouseover?

Reply:
Yes, that is me. Sorry. Full name is now in the profile.